bake/bake.go

```diff
@@ -144,6 +144,9 @@ func ReadTargets(ctx context.Context, files []File, targets, overrides []string,
 	// The logic is purposely duplicated from `build/build`.go for keeping this visible in `bake --print`.
 	if v := os.Getenv("SOURCE_DATE_EPOCH"); v != "" {
 		for _, f := range m {
+			if f.Args == nil {
+				f.Args = make(map[string]*string)
```

Comment:
I think it's only the assignment to the map that causes the panic, the if condition is valid even if f.Args is nil - maybe we should only create a new map inside the condition?

Reply:
Not a huge fan of dereferencing the nil slice even if it is valid. Just my preference though.

Reply:
SGTM